src/server/service/kv.rs

```diff
@@ -52,6 +57,316 @@ pub struct Service<T: RaftStoreRouter + 'static, E: Engine, L: LockMgr> {
     snap_scheduler: Scheduler<SnapTask>,
 
     thread_load: Arc<ThreadLoad>,
+
+    // timeout set to 0 means batch is collected without crossing commands
```

Comment:
Please use another config to present whether cross command.

Reply:
Why not use `Duration`?